src/layouts/Account-HEDA Organization Layout.layout

```diff
@@ -148,6 +148,12 @@
         <fields>GPA__c</fields>
         <relatedList>Program_Enrollment__c.Account__c</relatedList>
     </relatedLists>
+    <relatedLists>
```

Comment:
Just to make sure, have we decided to have related list?

Reply:
We should probably chat here. I could have sworn we had a discussion in the HEDA chat where Kevin Swiggum suggested we always add the new related lists, etc. Since during upgrades they don't get pushed, there's no worry in overwriting what customers have.